susemanager/bin/migrate-cobbler.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
This script will run until the end even if there are errors at any step, which is fine as long as this is really what you want.

Reply:
Yes, there is next to no chance to re-start in case of failure. A few of the steps might not be critical, but in general you will end up with a totally hosed system when eg. "zypper dup" runs into any problem.